go/teams/loader.go

```diff
@@ -332,9 +332,11 @@ func (l *TeamLoader) load1(ctx context.Context, me keybase1.UserVersion, lArg ke
 		return nil, nil, fmt.Errorf("team loader fault: got nil from load2")
 	}
 
-	// Only public teams are allowed to be behind on secrets.
-	// This is allowed because you can load a public team you're not in.
-	if !l.hasSyncedSecrets(mctx, ret.teamShim()) && !ret.team.Chain.Public {
+	// Public teams are allowed to be behind on secrets since you can load a
+	// public team you're not in. Bot members don't have any secrets and are
+	// also exempt.
+	if !l.hasSyncedSecrets(mctx, ret.teamShim()) &&
+		!(ret.team.Chain.Public || ret.team.Chain.UserRole(me).IsBot()) {
```

Comment:
Makes sense. But you can ditch this paragraph if you want. External callers will need to handle secretless teams because of bots anyhow.